src/test/java/io/confluent/connect/jdbc/sink/PreparedStatementBinderTest.java

```diff
@@ -66,7 +77,18 @@ public void bindRecord() throws SQLException {
         .put("float", (float) 2356.3)
         .put("double", -2436546.56457)
         .put("bytes", new byte[]{-32, 124})
-        .put("age", 30);
+        .put("age", 30)
+        .put("decimal", new BigDecimal("1.5").setScale(0, BigDecimal.ROUND_HALF_EVEN));
+
+    Calendar calendar = Calendar.getInstance();
+    calendar.setTimeInMillis(0);
+    calendar.set(2000, 1, 1);
+    valueStruct.put("date", calendar.getTime());
```

Comment:
It should be cleaner to just instantiate `Date` directly here and a few lines below, e.g. `new Date(long)` and not mess with the `Calendar` instance.